byzcoin/bcadmin/main.go

```diff
@@ -13,12 +13,17 @@ import (
 	"github.com/dedis/cothority/byzcoin/bcadmin/lib"
 	"github.com/dedis/cothority/darc"
 	"github.com/dedis/cothority/darc/expression"
+	"github.com/dedis/cothority/skipchain"
 	"github.com/dedis/kyber/util/random"
 	"github.com/dedis/onet"
 	"github.com/dedis/onet/app"
 	"github.com/dedis/onet/cfgpath"
 	"github.com/dedis/onet/log"
 	"github.com/dedis/onet/network"
+
+	"encoding/json"
+
+	"github.com/qantik/qrgo"
```

Comment:
what's the reason for using this library instead of something that's more well-known like https://github.com/skip2/go-qrcode

Reply:
We were already using it in cisc, so I asked @LMeynent to not introduce another dependency. Also, qrgo is interesting because it writes to the terminal, instead of requiring you to open a PNG.